SafeNeighbourhood/Pages/Index.cshtml.cs

```diff
@@ -15,11 +16,22 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
+                //Get Summary Neighborhood Cleanliness Data
```

Comment:
This is a brand new Json stream I created on data.cincinnati-oh.gov, of a summary Total of Trash/Litter/Grass Complaints by Neighborhood.  Had to create an account and save a custom view with a filter on Neighborhood and some other field, then published it to the site. The Json URL is accessible from the Export->SODA API tab.  Copy and paste the endpoint from there into the C# code.